packages/webvh/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@credo-ts/webvh",
+  "version": "0.5.16",
+  "description": "Credo WebVH DID method implementation",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "files": [
+    "build"
+  ],
+  "publishConfig": {
+    "main": "build/index",
+    "types": "build/index",
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "pnpm run clean && pnpm run compile",
+    "clean": "rimraf -rf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "pnpm run build",
+    "test": "jest",
+    "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand --config jest.config.js"
+  },
+  "dependencies": {
+    "@credo-ts/anoncreds": "workspace:*",
+    "@credo-ts/askar": "workspace:*",
```

Comment:
I think we can also remove askar dependency now we are using Agent's wallet, right?

Reply:
yep good call! thanks